src/sweeper/sweepers/empties.py

```diff
@@ -24,16 +24,16 @@ def sweep(self):
         arcpy.env.workspace = self.workspace
 
         with arcpy.da.SearchCursor(self.table_name, fields) as search_cursor:
-            print("Looping through rows in '{}' ...".format(self.table_name))
+            #print("Looping through rows in '{}' ...".format(self.table_name))
```

Comment:
remove the print statements or revert the commented out code please.

Reply:
@eneemann are you okay if I remove the print statements, or do you want to handle that?  or do you want to preserve them while debugging stuff?

Reply:
> @eneemann are you okay if I remove the print statements, or do you want to handle that? or do you want to preserve them while debugging stuff?

Yeah, that's fine.  You can remove them.

Reply:
done.